src/test/java/org/folio/pv/api/PasswordValidatorControllerApiTest.java

```diff
@@ -1,8 +1,6 @@
 package org.folio.pv.api;
 
-import static org.apache.http.HttpStatus.SC_BAD_REQUEST;
-import static org.apache.http.HttpStatus.SC_NOT_FOUND;
-import static org.apache.http.HttpStatus.SC_OK;
+import static org.apache.http.HttpStatus.*;
```

Comment:
Please not use '*' in imports.

Also, your import is not in proper order. Please, define these rules in your IDE and fix imports in all classes that you changed.

import static all other
empty line
import static org.folio.*
empty line
import java/javax.*
empty line
import all other
empty line
import org.folio.*

Reply:
Big thanks, Pavel!